pages/common/od.md

```diff
@@ -0,0 +1,25 @@
+# od
+
+> Display a file contents in hexadecimal, octal, or printable character format.
```

Comment:
"Display file contents", as you originally had it, worked well. If you prefer to add "a", then the wording should be "Display a file's contents", otherwise the sentence isn't grammatically correct, I'm afraid.